.github/workflows/maven-publish-scala.yml

```diff
@@ -58,8 +63,8 @@ jobs:
               }
           },{
             "id": "ossrh",
-            "username": "${{ secrets.OSSRH_USERNAME }}",
-            "password": "${{ secrets.OSSRH_PASSWORD }}",
+            "username": "${OSSRH_USERNAME}",
```

Comment:
put this in 
env:

like

 - name: Do something
      env:
        FOO_TOKEN: ${{ secrets.FOO_TOKEN }}
      run: |
        echo "FOO_TOKEN = ${FOO_TOKEN}"

Reply:
> put this in env:
> 
> like
> 
> * name: Do something
>   env:
>   FOO_TOKEN: ${{ secrets.FOO_TOKEN }}
>   run: |
>   echo "FOO_TOKEN = ${FOO_TOKEN}"

test:
~~~
- name: Set up OSSRH
      env:
        OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
        OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
      run: |
        echo "OSSRH_USERNAME = ${OSSRH_USERNAME}"
        echo "OSSRH_PASSWORD = ${OSSRH_PASSWORD}"
~~~
still can't get it
~~~
Run echo "OSSRH_USERNAME = ${OSSRH_USERNAME}"
OSSRH_USERNAME = 
OSSRH_PASSWORD = 
~~~